inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/softmax/softmax_kernel_items_class_optimized.cpp

```diff
@@ -60,15 +60,34 @@ SoftmaxKerneItemsClassOptimized::Parent::DispatchData SoftmaxKerneItemsClassOpti
 
     dispatchData.leftovers = item_class_count % workitems_per_classes;
 
-    if (item_class_count >= 32) {
-        dispatchData.efficiency = FORCE_PRIORITY_7;
-    } else {
-        dispatchData.efficiency = DONT_USE_IF_HAVE_SOMETHING_ELSE;
-    }
-
     return dispatchData;
 }
 
+KernelsPriority SoftmaxKerneItemsClassOptimized::GetKernelsPriority(const Params& params, const optional_params& /*options*/) const {
+    const auto& p = static_cast<const softmax_params&>(params);
+    auto& input = p.inputs[0];
+
+    size_t item_class_count = 0;
```

Comment:
Please reuse code for `item_class_count ` calculation across  `SetDefault`  and `GetKernelsPriority` methods

Reply:
Thanks, fixed